uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -361,10 +361,12 @@ define([
         if (missingRequiredFields.length) {
           var notification = CRM.alert(missingRequiredFields.join(', '),
             missingRequiredFields.length === 1 ? 'Required field' : 'Required fields', 'error');
+
           $timeout(function () {
-            notification.close();
+            notification && notification.close();
```

Comment:
The whole `$timeout` should not be called if `notification` is not present, so a if condition can be added before the $timeout